open-metadata-implementation/common-services/generic-handlers/src/main/java/org/odpi/openmetadata/commonservices/generichandlers/OpenMetadataAPIGenericHandler.java

```diff
@@ -10835,6 +10845,7 @@ public List<EntityDetail> getEntitiesByValue(String       userId,
      * @param specificMatchPropertyNames list of property value to look in - if null or empty list then all string properties are checked.
      * @param exactValueMatch indicates whether the value must match the whole property value in a matching result, or whether it is a
      *                        RegEx partial match
+     * @param caseInsensitive set to true to have a case-insensitive exact match regular expression
```

Comment:
From this javadoc It seems there are 3 permutations 
- exact match case insensitive
- exact match not case sensitive
- not exact match and the case insensitive flag is ignored.
 
I assume all 4 permutations are supported, if so we should remove the reference to exact match in this parameter description

Reply:
The purpose of the changes in generic handler is to acomodate the caseInsensitive param, as part of a bug fix (minimal intrusion, not changing the actual behaviour). The generic handler was ignoring it and the method in repositoryHelper was defaulting it to false (case sensitive).  By exposing caseInsensitive param, it allows callers to pass it. The asset catalog will pass its value coming from the user, the other are sending what was previously used: false.